api/handlers_metrics.go

```diff
@@ -204,6 +207,36 @@ func parseQuery(r *http.Request, request cloudwatch.MetricsRequest) error {
 	}
 	request["end"] = end
 
+	height := int64(400)
+	if h, ok := queries["height"]; ok {
+		hint, err := strconv.ParseInt(h[0], 10, 64)
+		if err != nil {
+			return errors.Wrap(err, "failed to parse height as int")
+		}
+
+		if hint > int64(2000) {
+			return fmt.Errorf("%d is greater than maximum height, 2000", hint)
```

Comment:
I'm used to `errors.New` but I guess this does the same thing

Reply:
Yeah, I think this is good where you want to use the `Sprintf` style expansion.  I wouldn't import `fmt` just for this, but its already there.